src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -16,6 +16,7 @@ internal class DecisionDagBuilder
         private readonly Conversions Conversions;
         private readonly TypeSymbol BooleanType;
         private readonly TypeSymbol ObjectType;
+        private HashSet<DiagnosticInfo> discardedUseSiteDiagnostics;
```

Comment:
>private HashSet<DiagnosticInfo> discardedUseSiteDiagnostics; [](start = 8, length = 60)

Do we have tests to cover all new code paths added in this file? The number of added tests seems small

Reply:
No. That remains on the open issue list, to be addressed after we have lowering and code gen implemented.